### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zbraniecki**
Update stages post March 2016 TC39 meeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ Proposals follow [this process document](https://tc39.github.io/process-document
 |---|---------------------------------------|--------------  | ------|------
 |   | [Intl.PluralRules][]                  | Caridy Patiño, Eric Ferraiuolo |     2 |
 |   | [Exposing Abstract Operations & Locale Info][]        | Zibi Braniecki  |     2 |
-|🚀 | [formatToParts][]                     | Zibi Braniecki |     2 |
```
Comment (caridy) [inline README.md]: can you move this to the first position?

File: README.md [hunk 2]
```diff
@@ -11,13 +11,13 @@ Proposals follow [this process document](https://tc39.github.io/process-document
...
+|   | [formatToParts][]                     | Zibi Braniecki |     3 |
 |   | [Fix 9.2.3 LookupMatcher algorithm][] | Rafael Xavier  |     0 |    
 |   | [Intl.NumberFormat round option][]    | Rafael Xavier  |     0 |
-|🚀 | [Intl.ListFormat][]                   | Zibi Braniecki |     0 |
```
Comment (caridy) [inline README.md]: can you move those above all proposals in stage 0?

**Comment by caridy** [comment]
once you reorder the proposals, I can merge it.

**Comment by zbraniecki** [comment]
updated

**Pull request merged**